.github/workflows/ci-macvim.yaml

```diff
@@ -128,6 +129,7 @@ jobs:
               --enable-rubyinterp=dynamic
               --enable-luainterp=dynamic
               --with-lua-prefix=/usr/local
+              --with-python-command=/usr/local/bin/python
```

Comment:
Is this necessary? Wouldn't it just figure out where Python 2 is?

Reply:
I think you should be changing `vi_cv_path_python` to `/usr/local/bin/python` intsead.

Reply:
I missed `vi_cv_path_python`... I'll fix it, thank you.